backend/etcd/etcd.go

```diff
@@ -162,8 +170,8 @@ func NewEtcd(config *Config) (*Etcd, error) {
 	// revision.
 	cfg.AutoCompactionMode = "revision"
 	cfg.AutoCompactionRetention = "2"
-	// Default to 4G etcd size. TODO: make this configurable.
-	cfg.QuotaBackendBytes = int64(4 * 1024 * 1024 * 1024)
+	cfg.QuotaBackendBytes = config.QuotaBackendBytes
```

Comment:
Do we need to do any validation for these values? Curious what the implications are if an arbitrary/random number is selected.

Reply:
Here's a good explanation about this limit here: https://github.com/etcd-io/etcd/issues/7045#issuecomment-268586740

If you configure it to use 12GB, etcd will print the following warning in the logs:
```
{"component":"etcd","level":"warning","msg":"backend quota 12884901888 exceeds maximum recommended quota 8589934592","pkg":"etcdserver","time":"2019-05-17T09:13:41-04:00"}
```

If you use a much lower value, sensu-backend will reach the quota real fast and errors will show up in the logs about `database space exceeded` or something like this.

I don't feel like we need to validate values passed directly to etcd but I'm okay if you'd prefer to!